### ISSUE TITLE
Per-OSD dashboard should include read and write latency

### ISSUE THREAD

**Issue by dillaman** [ceph/cephmetrics#207]
Something like the following:

quantile($percentile / 100, ceph_osd_op_r_latency_sum / ceph_osd_op_r_latency_count * 1000)
quantile($percentile / 100, ceph_osd_op_w_latency_sum / ceph_osd_op_w_latency_count * 1000)

It might also be nice to have a global latency view on the homepage (perhaps merged w/ the apply vs commit latency graph?).

**Comment by pcuzner** [comment]
per osd read/write is a good idea. Need to think about the best place to put it though. The ceph osd information dashboard picks up all osd's so in a bigger cluster that'll be a mess. Maybe a percentile view would work under OSD Summary, and add the per osd r/w down into the filestore and bluestore sections that makes use of the osd id pulldown?

**Comment by dillaman** [comment]
Yeah — even if it’s just a top-k high latency graph w/ the ability to drill down into OSDs by host or name if needed perhaps.

**Comment by sogabe** [comment]
It would be nice to have views of both per OSD r/w and top-k high latency graph.

I've quick implemented the per OSD view on the ceph-overview for PoC. It seems very insightful information.

<img width="1221" alt="per-osd-op-latency-poc" src="https://user-images.githubusercontent.com/1396486/43507280-a122d9f4-95a7-11e8-9827-ddd131075fca.png">
